scripts/README.md

```diff
@@ -22,3 +22,7 @@
    ```sh
    scripts/ffigen_helper.sh <version>
    ```
+
+4. Update callbacks data
+  * Run `./generate_callbacks.sh verify` to check type substitution. Build errors will have to be addressed by editing `gen_callbacks.py` - if it's missing types errors - modify list of additional headers (EXTRA\_HEADERS), if substitution is not found or assert fails - edit the type mapping.
+  * Run `./generate_callbacks.sh` to update `callbacks.cc` with callbacks data.
```

Comment:
3-space indentation expected (**exactly 3**, not 2 or 4!)

Also where can I edit the type mapping?